### PR TITLE
[#1448] add the changelog entry

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 CHANGELOG
 =========
 
+* fixed a BC break that prevented the `CamelKeysNormalizer` from removing leading underscores
```
Comment (Ener-Getick) [inline CHANGELOG.md]: Maybe this should be entitled 1.7.9?

Comment (Ener-Getick) [inline CHANGELOG.md]: In a sub section i mean

Comment (xabbuh) [inline CHANGELOG.md]: good idea, done

**Comment by Ener-Getick** [comment]
Status: reviewed

**Pull request merged**